source/math/FloatRandomGenerator.ooc

```diff
@@ -65,12 +75,24 @@ FloatGaussianRandomGenerator: class extends FloatRandomGenerator {
 		this _backend free()
 		super()
 	}
+	setRange: func (=_mu, =_sigma) {
+		_hasSecond = false
```

Comment:
I would welcome the possibility to change the range of generated numbers for existing generator object.